packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -21,30 +21,33 @@ import BlockUserModal from 'Components/block-user/block-user-modal';
 import ErrorModal from 'Components/error-modal/error-modal';
 import classNames from 'classnames';
 import './advertiser-page.scss';
+import { OnlineStatusIcon, OnlineStatusLabel } from 'Components/online-status';
 
 const AdvertiserPage = () => {
     const { general_store, advertiser_page_store, buy_sell_store } = useStores();
 
+    const is_my_advert = advertiser_page_store.advertiser_details_name === general_store.advertiser_info.name;
+    const info = is_my_advert ? general_store.advertiser_info : advertiser_page_store.counterparty_advertiser_info;
     const {
         basic_verification,
         buy_orders_count,
         created_time,
         first_name,
         full_verification,
-        id,
+        is_online,
+        last_online_time,
         last_name,
         name,
         rating_average,
         rating_count,
         recommended_average,
         recommended_count,
         sell_orders_count,
-    } = advertiser_page_store.counterparty_advertiser_info;
```

Comment:
When you are viewing your own ads, this will return the previously viewed counterparty advertiser's info instead of your own info